sdk/servicebus/azure-servicebus/azure/servicebus/aio/_async_auto_lock_renewer.py

```diff
@@ -130,6 +130,8 @@ def register(
          Default value is None (no callback).
          :rtype: None
         """
+        if not isinstance(renewable, ReceivedMessage) and not isinstance(renewable, ServiceBusSession):
+            raise TypeError("AutoLockRenewer only supports registration of types azure.servicebus.aio.ReceivedMessage (via a receiver's receive methods) and azure.servicebus.aio.ServiceBusSession (via a session receiver's property receiver.Session).")
```

Comment:
nit: it's "session" not "Session"
```suggestion
            raise TypeError("AutoLockRenewer only supports registration of types azure.servicebus.aio.ReceivedMessage (via a receiver's receive methods) and azure.servicebus.aio.ServiceBusSession (via a session receiver's property receiver.session).")
```